README.md

```diff
@@ -3,17 +3,41 @@
 
 # What is this?
 
-Load BIOM tables and sample metadata from a lot of studies into Redis to facilitate search using observation IDs or metadata. redbiom is intended as a caching layer. The command line application, for many of the commands, supports Unix pipes allowing for a fair bit of composability. As a hypothetical example, let's say we had loaded a bunch of microbiome data picked againts Greengenes 13_8 at 97% (and into an appropriately named context). To obtain a BIOM table representing all samples in which either OTUs 367523 or 3064251 exist, you can perform the following set of querys:
+Redbiom is a cache service to sample metadata and sample data. It allows for rapidly:
 
-    $ redbiom search observations --context "greengenes-13_8-97" 367523 3064251 | redbiom fetch samples --from - --output result.biom
+* finding samples by the observations they contain
+* finding samples by arbitrary metadata searches
+* summarizing samples over metadata
+* retieval of sample data into BIOM
+* discovering metadata categories
+* pulling out sample data from different processing types (e.g., search over 16S, retrieve WGS)
 
-Additionally, assuming a comprehensive set of metadata are available, you can obtain summarized environment information:
-    
-    $ redbiom search observations --context "greengenes-13_8-97" 367523 3064251 | redbiom summarize samples --from - --category "env_biome"
+Redbiom is designed to handle biological and technical replicates. Specifically, it allows for a one to many relationship between a sample's metadata and its data, both within and between preparation types.
+
+This repository defines the de facto redbiom data representation, and one possible interface into the resource. Other interfaces (e.g., Javascript) are possible to define. Please see the Design section below for details about how other interfaces can be written.
+
+By default, redbiom will search against `qiita.ucsd.edu`. This can be changed at runtime by setting the `REDBIOM_HOST` environmental variable, e.g., `export REDBIOM_HOST=http://qiita.ucsd.edu:7379`. The default host is **read-only** and administrative functions like loading data will not work against it.
```

Comment:
Should we define a given port for qiita? This will allow us to redirect with nginx and avoid hitting the actual qiita workers.